src/sentry/conf/server.py

```diff
@@ -1452,66 +1452,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 # `features.manager.add()`. Defining feature defaults here is deprecated.
 # If you must add a feature here, please maintain alphabetical ordering
 SENTRY_FEATURES: dict[str, bool | None] = {
-    # potentially unused.
-    "organizations:api-keys": False,
```

Comment:
Could you add an explicit default=False for this one too?

Reply:
Sure, but I don't see this feature flag being used anywhere in sentry or getsentry.